scripts/menu.sh

```diff
@@ -100,6 +100,10 @@ function setup() {
 
   export AWS_ACCESS_KEY_ID=$(inputbox "Amazon Web Services" "Access Key ID" "${AWS_ACCESS_KEY_ID}")
   export AWS_SECRET_ACCESS_KEY=$(inputbox "Amazon Web Services" "AWS Secret Key" "${AWS_SECRET_ACCESS_KEY}")
+<<<<<<< Updated upstream
+=======
+  export AWS_S3_BUCKET=$(inputbox "Amazon Web Services" "AWS S3 Bucket Name" "${AWS_S3_BUCKET}")
```

Comment:
Only dangerous thing with this is bucket names need to be globally unique. Users have trouble picking names. =)